fec.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               fec
-version:            0.2.0
+version:            1.1.1
```

Comment:
If we can keep the initialize call but make it a no-op we don't have to bump the major version, right?